pkg/handler/instrumented_enqueue_object.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
Should we not cover it with the tests?

Reply:
@joelanford you did mention about checking for this metrics in e2e tests. WDYT about unit tests?

Reply:
We will check in the if the project scaffold is exporting the metrics, however, we are adding here a pkg/lib feature so we IMO should cover with tests. Also, I think we agreed that the tests and docs are part of all tasks as well.

Reply:
We will check in the e2e if the project scaffolded is exporting the metrics, however, we are adding here a pkg/lib feature so we IMO it should be covered with tests.

Reply:
Should we be using gingko ?

Reply:
+1 to unit tests
+1 to ginkgo

Reply:
@joelanford @camilamacedo86 As discussed, will do a follow ip PR for adding testcases.